ui/src/app/components/runs/filters/number-range-filter/number-range-filter.component.html

```diff
@@ -0,0 +1,28 @@
+<!--
+  ~ Copyright 2018 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="clr-row">
```

Comment:
No offence, but the filter looks a bit ugly :) The trash button should be aligned with the input and the close button imo. I'm fine with postponing it, because these things can get very time-consuming

Reply:
Noted. Will take a look in next "sprint".